pkgs/servers/home-assistant/default.nix

```diff
@@ -99,10 +100,10 @@ in with py.pkgs; buildPythonApplication rec {
     aiohttp astral async-timeout attrs bcrypt certifi ciso8601 httpx jinja2
     pyjwt cryptography pip python-slugify pytz pyyaml requests ruamel_yaml
     setuptools voluptuous voluptuous-serialize yarl
-    # From default_config. frontend, http, image, mobile_app and recorder components as well as
-    # the auth.mfa_modules.totp module
-    aiohttp-cors defusedxml distro emoji hass-frontend pynacl pillow pyotp
-    pyqrcode sqlalchemy
+    # From default_config. dhcp, frontend, http, image, mobile_app and
+    # recorder components as well as the auth.mfa_modules.totp module
+    aiohttp-cors aiohue defusedxml distro emoji hass-frontend pynacl pillow pyotp
+    pyqrcode scapy sqlalchemy python-openzwave-mqtt
```

Comment:
Why were these three added? Scapy makes sense for dhcp I guess, but the rest?

Reply:
`aiohue` was required when I reenabled `test_loader.py` and `python-openzwave-mqtt` was required for testing `config`, see https://github.com/NixOS/nixpkgs/pull/111785#issuecomment-772784234.

Reply:
Then they should be `checkInputs`.

Reply:
Oh right. That goes for a few more of the deps I guess. Not sure what I was thinking.

Reply:
Addressed in #111921